Library/Homebrew/override.rb

```diff
@@ -0,0 +1,53 @@
+# typed: true
+# frozen_string_literal: true
+
+module Homebrew
+  # Helper module for overriding machine-specific methods.
+  #
+  # @api private
+  class Override
+    class << self
+      extend T::Sig
+
+      ARCH_OPTIONS = [:intel, :arm64].freeze
+      OS_OPTIONS = [:macos, :linux].freeze
```

Comment:
Can/should these shared with `on_system.rb`?

Reply:
Actually, these aren't used anywhere so I'm just going to delete them 😅